scripts/mysqld_safe.sh

```diff
@@ -21,9 +21,11 @@ mysqld_ld_library_path=
 flush_caches=0
 numa_interleave=0
 wsrep_on=0
+simulate=0
```

Comment:
I'm unsure if this justification is sufficient. Let me know if you want this changed. Thanks for mentioning systemd as a user, I might be able to write a socket activation test case with the user mode.